examples/demo.rs

```diff
@@ -26,7 +30,8 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     {
         "isActive": "True",
         "numAttempts": "",
-        "grade": "81"
+        "grade": "81",
+        "notes": ""
```

Comment:
I'm unsure here: should empty strings not be deserialized to `Some(String::new())`?

Is there something that cannot be unambiguously deserialized into a String? Only that should be `None`.